openquake/server/static/css/engine.css

```diff
@@ -65,7 +65,7 @@ h2 {
 }
 
 .nav-logo {
-    background: url(/static/img/oq-logo-engine.png) no-repeat;
+    /* background: url(/static/img/oq-logo-engine.png) no-repeat; */
```

Comment:
Why did you comment out the logo?

Reply:
Because, to be able to use dynamic {{ STATIC }} path prefix templating I moved it in the html template scope.